server/README.md

```diff
@@ -0,0 +1,22 @@
+# TACC Frontera Web Portal - Server
+
+## Directories
+
+- _`cms`: Isolate assets for the CMS website._
+- `conf`: Configure services (like CMS, Nginx, and Docker that runs them).
+- _`docs`: Isolate assets for the Tech Docs / User Guide._
+- _`media`: Dumped assets for the temporary CMS in which the Portal runs._
+- `portal`: Isolate code for the Portal.
+- _`static`: Dumped assets for the temporary CMS in which the Portal runs._
+
+Any _`italic`_ directories are dynamically created, thus should be ignored by version control.
+
+## Notes
+
+### `cms/media|static` vs. `./media|static`
+
+The `cms/media` & `cms/static` directories are read by the CMS that creates the website (and shared header). During local CMS development, you may update these directories to reflect CMS changes to the Portal.
+
+You may learn more about [How To Have Local Portal Use Local CMS](https://confluence.tacc.utexas.edu/x/OoC2C).
+
+The `./media` & `./static` directories are read by the _temporary_ CMS scaffold that hosts the Portal client app. They should not be manipulated during runtime. They will become moot when the Portal client app is hosted without a CMS scaffold.
```

Comment:
__Please, Help!__: Is this accurate? _I think now (with my greater knowledge) that the contents of these directories might be updated during runtime by an automatic `collectstatic` performed by an automatic `runserver` (as is done for the CMS in a dev environment)._

Reply:
Yes, mostly accurate. `media` and `static` are filled during `collectstatic`, manually

Reply:
Thank you. I have removed some text (https://github.com/TACC/Frontera-Portal/pull/60/commits/766ad2831e67d0fa5bc1d2aac6c480c31ffbc00f), so that what remains is (hopefully) objective, minimal, and factual.